profiles/base.cfg

```diff
@@ -208,31 +212,14 @@ recipe = collective.recipe.omelette
 eggs = ${instance:eggs}
 ignores = rdflib
 
-[ipzope]
```

Comment:
Any particular reason why you removed ipzope?

Reply:
actually, no. It was at a point where I wanted to create a personal buildout.
should I re-add it?

Reply:
I've done so locally, I'll commit that. Cheers!